src/keys-builder/utils/create-translation.ts

```diff
@@ -0,0 +1,59 @@
+import { unflatten } from 'flat';
+import { po } from 'gettext-parser';
+
+import { getConfig } from '../../config';
+import { mergeDeep, stringify } from '../../utils/object.utils';
+
+function handleReplace(currentTranslation, translation, replace) {
```

Comment:
```suggestion
function resolveTranslation({currentTranslation, translation, replace}) {
```
We might have more logic here in the future.